components/ServiceDetails/Sections/Offers.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="offers">
+    <el-table
+      :data="offers"
+      style="width: 100%">
+      <el-table-column
+        prop="index"
+        label="Index"
+        width="180">
+      </el-table-column>
+      <el-table-column
+        prop="price"
+        label="Price">
+      </el-table-column>
+      <el-table-column
+        prop="duration"
+        label="Duration">
+      </el-table-column>
+      <el-table-column
+        prop="status"
+        label="Status">
+          <template slot-scope="scope">
+            <div class="status" :title="scope.row.status ? 'active' : 'passive'">
+              <status-indicator :positive="scope.row.status"></status-indicator>
```

Comment:
The property in the offer is `active` and not `status`.
I added an offer and disabled it but i didn't see any change in the ui.
https://deploy-preview-4--mesg-marketplace.netlify.com/services/0x6b7102dbfeb4332cbbdd5684f43592f5b91203037a1abc183d52dccc903c71ae#offers

Reply:
done